### PR TITLE
Fixes #78 - websearch support for chat client

### LINKED ISSUE #78: Implement web result for unknown responses.
For the responses like **I don't know how to answer this. Here is a web search result:** show the web result or a generate a hyperlink on the query.
issue1

### PR DESCRIPTION
**Pull request by uday96**
**Demo Link:** http://susiwebsearch.surge.sh/
**Sample Queries:** 'Germany', 'Marvel', 'artificial Intelligence' etc which give action type websearch

For few queries like 'find me restaurants', even duckduckgo doesnt give any response. How to handle such cases?

Also the css can be changed. And the number of results too.
Right now, Im displaying all results without 'Name' attribute i.e popular results and one from each category (when 'Name' attribute is present).

ai

I refactored MessageListItem.js as I added actions attribute to message. I know its not related to this issue, will take care from next time :)

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -60,21 +62,50 @@ export function createSUSIMessage(createdMessage, currentThreadID) {
...
+          let actionIndex = actions.indexOf('websearch');
+          let query = response.answers[0].actions[actionIndex].query;
+          $.ajax({
+            url: 'http://api.duckduckgo.com/?format=json&q='+query,
+            dataType: 'jsonp',
+            crossDomain: true,
+            timeout: 3000,
+            async: false,
+            success: function (data) {
+                receivedMessage.websearchresults = data.RelatedTopics;
```
Comment (rishiraj824) [inline src/actions.js]: Can we have the Abstract to be the first card with the image. And then the RelatedTopics? @uday96

Comment (uday96) [inline src/actions.js]: Sure, that would be more informative. Will make changes :+1:

**Comment by uday96** [comment]
@rishiraj824 Added the abstract if present as the first tile. But the css for the card is not complete. I hardcoded the height as 150px. Must make it auto size and render the contents accordingly.

**Comment by uday96** [comment]
Messed up while squashing commits :sob: 
PR at https://github.com/fossasia/chat.susi.ai/pull/111

**Pull request closed without merging**